konfera/register/urls.py

```diff
@@ -19,7 +19,8 @@
 
 
 urlpatterns = [
-    url(r'^volunteer/event/(?P<slug>[\w, -]+)/$', views.register_volunteer, name='event_register_volunteer'),
-    url(r'^event/(?P<event_slug>[\w, -]+)/ticket/(?P<ticket_uuid>[\w, -]+)/$', views.private_registration,
+    url(r'^event/(?P<slug>[\w, -]+)/publicticket/(?P<ticket_uuid>[\w, -]+)/$', views.public_registration,
```

Comment:
we dont need publicticket/privateticket in url as you have uuid which identifies ticket exactly and you can see in ticket what type it is.

My suggestion is to have  urls:
/event/<slug>/<uuid> - this way doesnt matter if public or private as uuid for private ones is not published anywhere on the site. We can send registration urls by email, etc. for private registrations.
/event/<slug>/volunteers - display volunteer form only if such ticket_type is available and is just one otherwise redirect to views.event_public_tickets
/event/<slug>/press  - display press form only if such ticket_type is available and is just one otherwise redirect to views.event_public_tickets
/event/<slug>/financial_aid  - display press form only if such ticket_type is available and is just one otherwise redirect to views.event_public_tickets